accounts/CHANGELOG.md

```diff
@@ -0,0 +1,444 @@
+
+
+* Change - Set user role on builtin users: [#102](https://github.com/owncloud/ocis/accounts/pull/102)
+
+   We now set the default `user` role on our builtin users.
+
+   https://github.com/owncloud/ocis/accounts/pull/102
+
+
+* Change - Add new builtin admin user: [#102](https://github.com/owncloud/ocis/accounts/pull/102)
+
+   We added a new builtin user `moss` and assigned the admin role.
+
+   https://github.com/owncloud/ocis/accounts/pull/102
+
+
+* Change - We make use of the roles cache to enforce permission checks: [#100](https://github.com/owncloud/ocis/accounts/pull/100)
+
+   The roles cache and its cache update middleware are used to make permission checks possible.
+   The permission checks take place in the accounts handler.
+
+   https://github.com/owncloud/ocis/accounts/pull/100
+
+
+* Change - We make use of the roles manager to enforce permission checks: [#108](https://github.com/owncloud/ocis/accounts/pull/108)
+
+   The roles cache and its cache update middleware have been replaced with a roles manager in
+   ocis-pkg/v2. We've switched over to the new roles manager implementation, to prepare for
+   permission checks on grpc requests as well.
+
+   https://github.com/owncloud/ocis/accounts/pull/108
+   https://github.com/owncloud/ocis-pkg/pull/60
+
+
+* Enhancement - Add create account form: [#148](https://github.com/owncloud/product/issues/148)
+
+   We've added a form to create new users above the accounts list.
+
+   https://github.com/owncloud/product/issues/148
+   https://github.com/owncloud/ocis/accounts/pull/115
+
+
+* Enhancement - Add delete accounts action: [#148](https://github.com/owncloud/product/issues/148)
+
+   We've added an action into the actions dropdown to enable admins to delete users.
+
+   https://github.com/owncloud/product/issues/148
+   https://github.com/owncloud/ocis/accounts/pull/115
+
+
+* Enhancement - Add enable/disable capabilities to the WebUI: [#118](https://github.com/owncloud/product/issues/118)
+
+   We've added batch actions into the accounts listing to provide options to enable and disable
+   accounts.
+
+   https://github.com/owncloud/product/issues/118
+   https://github.com/owncloud/ocis/accounts/pull/109
+
+
+* Enhancement - Improve visual appearance of accounts UI: [#222](https://github.com/owncloud/product/issues/222)
+
+   We aligned the visual appearance of the accounts UI with default ocis-web apps (full width,
+   style of batch actions), added icons to buttons, extracted the buttons from the batch actions
+   dropdown into individual buttons, improved the wording added a confirmation widget for the
+   user deletion and removed the uid and gid columns.
+
+   https://github.com/owncloud/product/issues/222
+   https://github.com/owncloud/ocis/accounts/pull/116
+
+# Changelog for [0.4.1] (2020-08-27)
+
+The following sections list the changes in ocis-accounts 0.4.1.
+
+[0.4.1]: https://github.com/owncloud/ocis/accounts/compare/v0.3.0...v0.4.1
+
+## Summary
```

Comment:
Codacy found an issue: [[no-duplicate-headings] Do not use headings with similar content (7:1)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)